documentation/templates/theme/initiatives/epics/stories/story_template.md

```diff
@@ -12,6 +12,100 @@ The detailed view of the order includes the list of items purchased, the quantit
 The order history page can be accessed from the customer account dashboard.
 The order history page is only accessible to authenticated customers.
 
+User Stories 1 for Epics 1:
```

Comment:
include links to the epics and/or references to which epics these user stories refer to